### ISSUE TITLE
Refine snow on top of slabs

### ISSUE THREAD

**Issue by Hirachosan** [BTW-Community/BTW-Public#227]
Currently make a full stack sticking out weirdly, should probably be half a stack

Also has rendering issues with groundcover next to it

**Comment by ammoore00** [comment]
Fixed invisible sides. Still renders too dark, and could use other tweaks.

**Comment by Hirachosan** [comment]
Yeah my idea was to make it only able to grow half the size or 1 taller and let the rest of the snow count it correctly from is visual size (slab+ snow layers)
